libraries/core-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,175 +1,118 @@
 import * as React from 'react'
-import {
-  forwardRef,
-  useState,
-  HTMLAttributes,
-  ReactNode,
-  SVGProps,
-} from 'react'
+import { forwardRef, useRef, useState, HTMLAttributes } from 'react'
 import styled, { css } from 'styled-components'
-import type { CSSObject } from 'styled-components'
 import { spacingsTemplate, typographyTemplate } from '@utils'
-import { tooltip as tokens, Placement } from './Tooltip.tokens'
+import { usePopper } from '@hooks'
+import { tooltip as tokens } from './Tooltip.tokens'
 
-const Wrapper = styled.div`
-  position: relative;
-  display: inline-flex;
-  justify-content: center;
-`
-
-type WrapperProps = {
-  top: string | number
-  bottom: string | number
-  right: string | number
-  left: string | number
-  transform: string
-}
+const Arrow = styled.div`
+  &,
+  &::before {
+    position: absolute;
+    width: 8px;
+    height: 8px;
+    z-index: -1;
+  }
 
-const StyledTooltipWrapper = styled.div<WrapperProps>`
-  ${({ top, bottom, right, left, transform }) =>
-    css`
-      bottom: ${bottom};
-      top: ${top};
-      right: ${right};
-      left: ${left};
-      transform: ${transform};
-    `}
-  position: absolute;
-  align-self: center;
-  z-index: 350;
-  white-space: nowrap;
-  ::after {
+  &::before {
     content: '';
+    transform: rotate(45deg);
+    background: #333;
   }
 `
 
-const StyledTooltip = styled.div`
+const StyledTooltip = styled.div<Pick<TooltipProps, 'open'>>`
   ${typographyTemplate(tokens.typography)}
   ${spacingsTemplate(tokens.spacings)}
+  z-index: 350;
```

Comment:
Doesn't popper handle z-indexes?

Reply:
We should still follow our own z-index hierarchy here https://eds-storybook-react.azurewebsites.net/?path=/story/documentation-z-index--z-index